scripts/lib/eksctl.sh

```diff
@@ -76,7 +76,7 @@ eksctl::init() {
 #   node_count          EKS cluster's node count
 #   cluster_kubeconfig  filename of cluster's kubeconfig
 #
-# sample: eksctl::create_cluster awesome-cluster us-west-2 1.18 m5.xlarge 4 awesome-cluster.kubeconfig
+# sample: eksctl::create_cluster awesome-cluster us-west-2 1.19 m5.xlarge 4 awesome-cluster.kubeconfig
```

Comment:
Unrelated, simply trying to make comment aligned with actual ClusterConfig
- https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/v2.4.2/scripts/lib/eksctl_tmpl_cluster.yaml